### PR TITLE
[MM-58059] Remove PostPriority feature flag

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
The feature flag didn't impact the server side anyway and only controlled the visibility of the system console setting. This PR removes it to clarify that the config setting controls the functionality.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-58059

#### Release Note
```release-note
Remove PostPriority feature flag
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/admin_definition.tsx [hunk 1]
```diff
@@ -2549,7 +2549,6 @@ const AdminDefinition: AdminDefinitionType = {
                             },
                             help_text_markdown: false,
                             isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.SITE.POSTS)),
-                            isHidden: it.configIsFalse('FeatureFlags', 'PostPriority'),
```
Comment (marianunez) [inline webapp/channels/src/components/admin_console/admin_definition.tsx:2552]: Seems that the feature flag was hiding the overall ability to enable the feature in the System Console and it was defaulted to `false` out of the box? I created a test server but this was available so I'm not sure where is this feature flag is being set.

**Review by marianunez — APPROVED** [review]

File: webapp/channels/src/components/admin_console/admin_definition.tsx (hunk 1 above)
Comment (hanzei) [reply webapp/channels/src/components/admin_console/admin_definition.tsx:2552]: > Seems that the feature flag was hiding the overall ability to enable the feature in the System Console and it was defaulted to `false` out of the box?

Yes, by default, the settings were hidden in the System Console.

I'm not sure why the setting showed up on the test server. Maybe they use some default feature flags?

**Review by calebroseland — APPROVED** [review]

**Comment by hanzei** [comment]
@yasserfaraazkhan Could you please QA review the PR? You can find the test steps in the JIRA ticket.

**Review by yasserfaraazkhan — APPROVED** [review]
tested toggling the flag and with posting messgae with different priority tags.
Working as expected.

**Pull request merged**